frontend/src/components/CommentListSimple.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {useEffect, useState} from 'react';
+import {Typography} from "@mui/material";
+import moment from "moment/moment";
+import axios from "axios";
+import {CommentModel} from "../model/CommentModel";
+import {BpmnDiagramModel} from "../model/BpmnDiagramModel";
+
+type CommentListProps = {
+    diagram: BpmnDiagramModel
+}
+
+function CommentListSimple(props: CommentListProps) {
+
+    const [commentList, setCommentList] = useState<CommentModel[]>([])
```

Comment:
I think you can omit the [] as initial value since you fill it with a useEffect

Reply:
If i omit this, the List is may be undefined...